src/bundle/Resources/public/js/scripts/admin.trash.list.js

```diff
@@ -1,4 +1,4 @@
-(function () {
+(function(Translator) {
```

Comment:
Can you update the convention here?
It should be `(function(global, doc, eZ, Translator) {`